### ISSUE TITLE
Please provide media upload error details

### ISSUE THREAD

**Issue by c-holtermann** [gramps-project/gramps-web#445]
I have had a hard time uploading my media data to the web. I had a lot of problems with media files. In the beginning none of them would upload. Only after a long time I realized that that was due to absolute paths (all my files had absolute paths). It would be helpful to show error messages in that case, something like "image {path} failed to upload because of ...". It would be helpful if the upload process would continue and present a list in the end with all the failed files and the reasons, maybe an option to retry. That makes it easier to figure out the different reasons. Another reason is files that are too big (and get stuck in a proxy). That in itself is not an issue of the gramps web app but can still happen and would be nice to mention. Error is 413. There might be different reasons for other errors: unstable internet connection, server overload...
I have put some debug statements into the upload code so I could see which errors arise at which files. Finally I have fixed all media issues and uploaded all files.
I find it very helpful that there now is a list of nonexistant media files in gramps web.
BTW it would be nice to have an abort button for the whole process at all stages.

Gramps 5.1.6
Gramps Web API 1.5.1
Gramps Web Frontend 24.1.2
locale: en
multi-tree: false
task queue: true

I realize this might be the wrong bug tracker (as this is mostly about the web sync tool)

**Comment by DavidMStraub** [comment]
> I realize this might be the wrong bug tracker (as this is mostly about the web sync tool)

This is the bug tracker of the web interface (frontend), not the sync tool.

What error was shown to you? When the upload failes, there is a red error symbol; hovering over it shows the error.

**Comment by c-holtermann** [comment]
I'll check.

**Comment by DavidMStraub** [comment]
Please let me know if you can provide more details, otherwise we can close this issue.

**Comment by DavidMStraub** [comment]
What do you mean by Gramps Desktop is stating there are 1306 missing images? Where does it state that?

**Comment by patvdleer** [comment]
The Gramps to web sync tool.

![image](https://github.com/user-attachments/assets/e3d5ec00-9bb7-45a5-9129-9ed4559612f0)

![image](https://github.com/user-attachments/assets/71a9b63a-3c40-4e99-ad69-b848c6553344)

**Comment by DavidMStraub** [comment]
Closing as off topic.
